### ISSUE TITLE
EnderIO conduits cant be broken on contraption

### ISSUE THREAD

**Issue by MaxBas** [ryanhcode/sable#207]
### Description

I was building with EnderIO conduits on a contraption. It worked fine at the beginning but at some point the conduits became indestructible. Whenever I assemble or disassemble the contraption the conduits also drop as items. Additionally they can't carry energy anymore. The logs state 
`[21:56:30] [Server thread/ERROR] [de.ry.sa.Sable/]: Failed to destroy old block during assembly BlockPos{x=20481029, y=125, z=20481045}
java.lang.NullPointerException: Cannot invoke "com.enderio.enderio.content.conduits.network.ConduitNodeImpl.detach()" because "node" is null`. 

At the end the game crashed once while trying to break a conduit.

### Game Log

https://mclo.gs/piFEeh5

**Comment by MaxBas** [comment]
After restarting the game (from the crash) the blocks are breakable again
